internal/types/infrastructure/infrastructure.go

```diff
@@ -61,26 +62,20 @@ func (i *Infrastructure) Execute() (interface{}, error) {
 	}
 }
 
-// updateJobStatus updates the job status (this es un placeholder - implementar con una DB real)
-func (i *Infrastructure) updateJobStatus(jobID, status string, result interface{}) {
```

Comment:
this wasn't being called anywhere. if we want to keep it we should use the nolint directive